### PR TITLE
[Doc] Adding blog post to model doc for `TimmWrapper`

### PR DESCRIPTION
**Pull request by ariG23498**
This PR adds the blog post link to the `TimmWrapper` documentation.

### CODE REVIEW

File: docs/source/en/model_doc/timm_wrapper.md [hunk 1]
```diff
@@ -47,6 +47,12 @@ Helper class to enable loading timm models to be used with the transformers libr
 >>> top5_probabilities, top5_class_indices = torch.topk(logits.softmax(dim=1) * 100, k=5)
 ```
 
+> [!NOTE]
+> For a more detailed overview please read the [official blog post](https://huggingface.co/blog/timm-transformers) on the timm integration.
```
Comment (qubvel) [inline docs/source/en/model_doc/timm_wrapper.md:51]: Maybe put it in `Resources` section as well?

**Review by qubvel — APPROVED** [review]
Thanks!

File: docs/source/en/model_doc/timm_wrapper.md [hunk 2]
```diff
@@ -47,6 +47,12 @@ Helper class to enable loading timm models to be used with the transformers libr
 >>> top5_probabilities, top5_class_indices = torch.topk(logits.softmax(dim=1) * 100, k=5)
 ```
 
+> [!NOTE]
```
Comment (stevhliu) [inline docs/source/en/model_doc/timm_wrapper.md:50]:
```suggestion
> [!TIP]
```

**Review by stevhliu — APPROVED** [review]
Thanks!

**Comment by ariG23498** [comment]
@stevhliu @qubvel I have taken care of the suggestions 🤗

**Review by qubvel — COMMENTED** [review]
Thanks, could you also add a pipeline tag and link to official image classification example + add "community" sign for you link, see example 

https://github.com/huggingface/transformers/blob/3998fa8aab73e78de9dd9717407c4108728e8f5b/docs/source/en/model_doc/rt_detr.md?plain=1#L74-L82

**Review by qubvel — APPROVED** [review]
Thanks!
